tests/js/client/shell/shell-cluster-agency.js

```diff
@@ -0,0 +1,439 @@
+/*jshint globalstrict:false, strict:false */
```

Comment:
Complete useless use of use. 
This test should be reworked in another way and be descoped from this PR.

Reply:
At least for now we should keep version with `"/_admin/execute"`. This and other tests which use this will be reworked/replaced/removed as part of standalone task